src/historical-data/historical-data.scss

```diff
@@ -34,45 +35,95 @@
       }
       li {
          min-width: 130px;
-         font-size: 14px;
+         font-size:90%;
          background: #2A3052;
          color: white;
       }
       .title {
-         background-image: url('../images/thin-arrowheads-pointing-down.svg') !important;
-         padding-left: 20px;
-         line-height: 34px;
-         background-size: 12px 12px !important;
-         background-position: 2px 50% !important;
-         background-repeat: no-repeat;
-         margin: 0;
-         white-space: nowrap;
-      }
-      .ui-state-active {
-         margin: 0 !important;
+
+        background-image: url('../images/thin-arrowheads-pointing-down.svg') !important;
+        padding-left: 7px;
+        line-height: 34px;
+        font-size:18px;
+
+        margin:auto;
+        background-size: 10px 10px !important;
+        background-position: 90px 50% !important;
+        background-repeat: no-repeat;
+        margin: 0;
+        white-space: nowrap;
       }
+
    }
    .ui-dialog-titlebar {
       .ui-dialog-title {
-         font-size: 16px;
+         
          width: 320px;
       }
    }
    .date-time {
+    
       position: absolute;
-      background: linear-gradient(#2A3052,#15212D);
-      height: 34px;
-      width: 210px;
-      left: 340px;
-      top: 0;
+      
+      
+      left: 360px;
+      top: 5px;
       color: white;
       input {
-         color: white;
+         
          position: relative;
-         background: transparent;
+         
          border: none;
          display: inline-block;
-         width: 100px;
+         width: 105px;
+      }
+      input:hover {
+        border: 0.2px solid #2a3052;
       }
    }
 }
+
+
+.ui-timepicker-hours{
+    padding:0;
+    background:white;
+}
+
+.ui-timepicker-minutes{
+    padding:0;
+    background:white;
+}
+
+.ui-corner-all {
+    border-radius: unset;
+}
+
+.ui-timepicker-title {
+    text-align: center;
+}
+
+.ui-timepicker td {
+    border: none;
+
+}
+
+.ui-timepicker td a {
+    display: block;
+    padding: .2em .3em;
+    width: 1.2em;
+    text-align: right;
+    text-decoration: none;
+    cursor: pointer;
+}
+
+.ui-timepicker-hours .ui-timepicker tbody {
+    border-right: 3px solid #f2f2f2;
+}
+
+.ui-widget-content button {
+    float: right !important;
+}
+
+.ui-datepicker-close {
+    display: none;
+}
```

Comment:
Perhaps we could move all these to be "under" .historical-data-dialog so they don't conflict with css in other windows